src/analysis/mellin_transform.lean

```diff
@@ -0,0 +1,333 @@
+        (rpow_le_rpow_of_exponent_ge ht h.le _),
+      rw [sub_re, one_re, sub_le_iff_le_add, sub_add_cancel],
+      rw [mem_ball_iff_norm', complex.norm_eq_abs] at hz,
+      have hz' := (re_le_abs _).trans hz.le,
+      rwa [sub_re, sub_le_iff_le_add, ←sub_le_iff_le_add'] at hz', } },
+  have h5 : integrable_on bound (Ioi 0),
+  { simp_rw [bound, add_mul, mul_assoc],
+    suffices : ∀ {j : ℝ} (hj : b < j) (hj' : j < a),
+      integrable_on (λ (t : ℝ), t ^ (j - 1) * (|log t| * ‖f t‖)) (Ioi 0) volume,
+    { refine integrable.add (this _ _) (this _ _),
+      all_goals { linarith } },
+    { intros j hj hj',
+      obtain ⟨w, hw1, hw2⟩ := exists_between hj,
+      obtain ⟨w', hw1', hw2'⟩ := exists_between hj',
+      refine mellin_convergent_of_is_O_scalar _ _ hw1' _ hw2,
+      { simp_rw mul_comm,
+        refine hfc.norm.mul_continuous_on _ is_open_Ioi,
+        refine continuous.comp_continuous_on continuous_abs (continuous_on_log.mono _),
+        exact subset_compl_singleton_iff.mpr not_mem_Ioi_self },
+      { refine (is_O_rpow_top_log_smul hw2' hf_top).norm_left.congr' _ (eventually_eq.refl _ _),
+        refine (eventually_gt_at_top 0).mp (eventually_of_forall (λ t ht, _)),
+        simp only [norm_smul, real.norm_eq_abs] },
+      { refine (is_O_rpow_zero_log_smul hw1 hf_bot).norm_left.congr' _ (eventually_eq.refl _ _),
+        refine eventually_nhds_within_iff.mpr (eventually_of_forall (λ t ht, _)),
+        simp only [norm_smul, real.norm_eq_abs] } } },
+  have h6 : ∀ᵐ (t : ℝ) ∂volume.restrict (Ioi 0), ∀ (y : ℂ),
+    y ∈ metric.ball s v → has_deriv_at (λ (z : ℂ), F z t) (F' y t) y,
+  { dsimp only [F, F'],
+    refine (ae_restrict_iff' measurable_set_Ioi).mpr (ae_of_all _ $ λ t ht y hy, _),
+    have ht' : (t : ℂ) ≠ 0 := of_real_ne_zero.mpr (ne_of_gt ht),
+    have u1 : has_deriv_at (λ z : ℂ, (t : ℂ) ^ (z - 1)) (↑t ^ (y - 1) * ↑t.log) y,
+    { convert ((has_deriv_at_id' y).sub_const 1).const_cpow (or.inl ht') using 1,
+      rw of_real_log (le_of_lt ht),
+      ring },
+    exact u1.smul_const (f t) },
+  simpa only [F', mellin, mul_smul] using
+    (has_deriv_at_integral_of_dominated_loc_of_deriv_le hv0 h1 h2 h3 h4 h5 h6).2,
+end
+
+/-- Suppose `f` is locally integrable on `(0, ∞)`, is `O(x ^ (-a))` as `x → ∞`, and is
+`O(x ^ (-b))` as `x → 0`. Then its Mellin transform is differentiable on the domain `b < re s < a`.
+-/
+lemma mellin_differentiable_at_of_is_O_rpow [complete_space E] [normed_space ℂ E]
+  {a b : ℝ} {f : ℝ → E} {s : ℂ}
+  (hfc : locally_integrable_on f $ Ioi 0)
+  (hf_top : is_O at_top f (λ t, t ^ (-a))) (hs_top : s.re < a)
+  (hf_bot : is_O (𝓝[Ioi 0] 0) f (λ t, t ^ (-b))) (hs_bot : b < s.re) :
+  differentiable_at ℂ (mellin f) s :=
+(mellin_has_deriv_of_is_O_rpow hfc hf_top hs_top hf_bot hs_bot).differentiable_at
+
+end mellin_diff
+
+section exp_decay
+
+/-- If `f` is `O(exp (-a * x))` at top for some `0 < a`, then it is `O(x ^ b)` for every `b`. -/
```

Comment:
Isn't there a better place for this lemma? If you find a suitable place but the file has already been ported, you can keep the lemma here but then add a `porting note: move to ... once the port is done`.

Reply:
I moved (a slight reformulation of) this lemma to `analysis.special_functions.pow`, which isn't ported yet.